### ISSUE TITLE
[vsdebug] "stopped" event needs a correct value for the "reason" field

### ISSUE THREAD

**Issue by PranayAgarwal** [facebook/hhvm#8135]
### HHVM Version
```
HipHop VM 3.25.0-dev (rel)
Compiler: 1519182417_090784734
Repo schema: ebd66af5d2ef02d9c0b602d944acbb6fc7c4b5f0
```
### Operating System and Version
* Ubuntu 16.04.3 LTS

### Problem

When HHVM sends a `stopped` event to VSCode, the `reason` is a user-readible string like "Run to location" or "Exception (exceptionName) thrown". According to the protocol, this needs to be one of a fixed set of values like 'step', 'breakpoint', 'exception', 'pause', 'entry' etc.

This string can be in the `description` or `text` fields instead.

See https://github.com/Microsoft/vscode-debugadapter-node/blob/master/protocol/src/debugProtocol.ts#L82

**Comment by ebluestein** [comment]
I have a fix in the pipeline for this, will have it out soon!

**Comment by ebluestein** [comment]
Fixed by 29692ab83eb16c3744bd0ba54178de4d8b3c10a9!
